.github/workflows/regression-tests.yml

```diff
@@ -30,7 +30,9 @@ jobs:
       - name: Set DEVPKG to pgver if unset
         if: ${{ env.DEVPKG == '' }}
         run: |
-          echo "DEVPKG=${{ matrix.pgver }}" >> $GITHUB_ENV;
+          # Cut off label and leave only major version number (17beta2 -> 17)
+          DEVPKG=$(echo ${{ matrix.pgver }} | sed 's/^\([0-9]\{2\}\).*/\1/')
+          echo "DEVPKG=$DEVPKG" >> $GITHUB_ENV;
```

Comment:
I'm not sure why this is needed. Wouldn't it be simpler to just list `17` in the matrix? This also reduces churn down the line when new betas come out.

Reply:
Ah, right. That's annoying. I think the way we handle this in pgAudit is better but it's not a trivial change to make.

Reply:
Seems so, it's differs not only in a way how to run tests, but also in repo branch structure. I think that change is not for this PR.

Reply:
Yeah, the branch structure would not be reproduced here, but the test containers could be. However, it doesn't make sense to hold up this commit for that. Your solution is fine.